dotcom-rendering/scripts/webpack/browser-targets.test.ts

```diff
@@ -11,7 +11,7 @@ describe('Browser targets are as expected', () => {
 			edge: '91.0.0',
 			firefox: '78.0.0',
 			ios: '10.3.0',
-			opera: '64.0.0',
+			opera: '88.0.0',
```

Comment:
Is this meant to be part of this PR?

Reply:
Yeah, for some reason reseting the package.lock means these are caught in failing tests.